src/db/mod.rs

```diff
@@ -282,6 +282,28 @@ impl Database {
         pipeline: impl IntoIterator<Item = Document>,
         options: Option<ChangeStreamOptions>,
     ) -> Result<ChangeStream> {
-        todo!();
+        let pipeline: Vec<Document> = pipeline.into_iter().collect();
+        let mut options = options;
```

Comment:
Ditto above about marking the parameter as `mut`

Reply:
fixed

Reply:
This doesn't seem to be fixed in the latest commit.

Reply:
didn't push earlier. fixed now